src/main/scala/tile/LazyRoCC.scala

```diff
@@ -357,7 +360,7 @@ class BlackBoxExampleModuleImp(outer: BlackBoxExample, blackBoxFile: String)(imp
                    ) ) with HasBlackBoxResource {
         val io = IO( new Bundle {
                       val clock = Input(Clock())
-                      val reset = Input(Bool())
+                      val reset = Input(Reset())
```

Comment:
[The reference RoCC is currently hard-coded to synchronous reset](https://github.com/chipsalliance/rocket-chip/blob/52843da3d8daaba5c32c488ee21d20aa2f640772/src/main/resources/vsrc/RoccBlackBox.v#L170).

I think this means that this should stay as `Bool` here or this needs to mix-in the `RequireSyncReset` trait.

Reply:
I had overlooked that! `RequireSyncReset` mixin sounds neat.

Reply:
> Thanks for doing this @kammoh! Generally, looks fine. Comments inline.
> 
> By chance have you tested this with any RoCCs? (By inspection, I see no issues.)

Thanks for the review! I had only tested `AccumulatorExample` and `CharacterCountExample`.  `AccumulatorExample` seemed to run the `chipyard` example program correctly. `CharacterCountExample` did not work correctly even before the change in this PR. It seems the current code only works when the target strings is aligned to cache line size, i.e. It works when adding `__attribute__ ((aligned (CacheBlockBytes)))` to `string`.